test/examples/examples.jl

```diff
@@ -2,6 +2,96 @@
 # but without checking the correctness of the solution.
 @testset verbose=true "Examples" begin
     @testset verbose=true "Fluid" begin
+        @trixi_testset "fluid/hydrostatic_water_column_2d.jl" begin
+            # import variables into scope
+            trixi_include(@__MODULE__,
+                          joinpath(examples_dir(), "fluid",
+                                   "hydrostatic_water_column_2d.jl"),
+                          fluid_system=nothing, sol=nothing, semi=nothing, ode=nothing)
+
+            hydrostatic_water_column_tests = Dict(
+                "(WCSPH) default" => (),
+                "(WCSPH) with source term damping" => (source_terms=SourceTermDamping(damping_coefficient=1e-4),),
+                "(WCSPH) with SummationDensity" => (fluid_density_calculator=SummationDensity(),
```

Comment:
```suggestion
                "WCSPH default" => (),
                "WCSPH with source term damping" => (source_terms=SourceTermDamping(damping_coefficient=1e-4),),
                "WCSPH with SummationDensity" => (fluid_density_calculator=SummationDensity(),
```
Why the parentheses?